core/commands/swarm.go

```diff
@@ -101,6 +100,9 @@ var swarmPeersCmd = &cmds.Command{
 			*/
 
 			if verbose || latency {
+				// set direction
+				ci.Direction = c.Stat().Direction
```

Comment:
Instead of overloading the `latency` flag, let's just add a `--direction` flag. I'm usually all for reducing the number of flags we have but tying latency to connection direction feels a bit weird.